pkgs/by-name/mo/moonlight-qt/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  darwin,
+  overrideSDK,
+  libsForQt5,
```

Comment:
It seems that they ship with qt6 by default almost everywhere, can we use it as well?

Reply:
That's an upstream issue most likely

Reply:
If you mean moonlight-qt upstream, we're not really doing anything special other than asking for precompiled QML files using the documented `CONFIG` option. https://github.com/moonlight-stream/moonlight-qt/blob/17a25484805ef927ad7c1b2906f885ee26eda928/app/app.pro#L13-L20

However, the QML precompilation is probably undesirable in the case of `CONFIG+=disable-prebuilts` since that is asking explicitly for Moonlight _not_ to internally bundle all the Moonlight dependencies. Using precompiled QML without bundling Qt could lead to the precompiled qmlc files being useless if the compile-time Qt version doesn't exactly match the runtime Qt version. I've [checked in a patch](https://github.com/moonlight-stream/moonlight-qt/commit/d73df12367749425b86b72c250bb0fba13ddfd29) to avoid this scenario, which may also allow you to migrate Moonlight to Qt 6 in NixOS.